win/enc_temp_folder/84b47e98f0e59dba8864125a99a69a5c/spell_scripts.cpp

```diff
@@ -0,0 +1,511 @@
+/* This file is part of the ScriptDev2 Project. See AUTHORS file for Copyright information
```

Comment:
Ahh :D Thats what happens if you alter a file in debug mode. Ill fix it afterwards.